File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/BeregningsInformasjonMinstenivatilleggPensjonistpar.java
package no.nav.domain.pensjon.regler.beregning2011;

import java.io.Serializable;

public class BeregningsInformasjonMinstenivatilleggPensjonistpar implements Serializable {


    /*
     * Beregnet pensjon pensjon
     */
    private double samletPensjon;
    /*
     * Minstepensjonsniva sats
     */
    private double mpnSatsOrdinaer;
    /*
     * Garantipensjonpensjonsniva sats
     */
    private double garPNSatsOrdinaer;
    /*
     * saertillegg sats
     */
    private double stSatsOrdinaer;
    /*
     * tt anvendt brukt i alderspensjon etter kapittel 19
     */
    private int tt_anv_AP;
    /*
     * tt anvendt brukt i alderspensjon etter kapittel 20
     */
    private int tt_anv_AP_Kapittel20;

    /*
     * tt anvendt brukt i uforepensjon
     */
    private int tt_anv_UP;
    /*
     * trygdetid i prorata beregning
     */
    private double prorataUP;
    /*
     * teller for proratabrok
     */
    private int prorataUPTeller;
    /*
     * nevner for proratabrok
     */
    private int prorataUPNevner;
    /*
     * personens gjeldende uttaksgrad
     */
    private int uttaksgrad;
    /*
     * personens gjeldende uforegrad
     */
    private int uforegrad;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/BeregningsInformasjonMinstenivatilleggPensjonistpar.kt
package no.nav.pensjon.regler.domain.beregning2011

import java.io.Serializable

class BeregningsInformasjonMinstenivatilleggPensjonistpar : Serializable {
    /*
     * Beregnet pensjon pensjon
     */
    var samletPensjon = 0.0

    /*
       * Minstepensjonsniva sats
       */
    var mpnSatsOrdinaer = 0.0

    /*
       * Garantipensjonpensjonsniva sats
       */
    var garPNSatsOrdinaer = 0.0

    /*
       * saertillegg sats
       */
    var stSatsOrdinaer = 0.0

    /*
       * tt anvendt brukt i alderspensjon etter kapittel 19
       */
    var tt_anv_AP = 0

    /*
       * tt anvendt brukt i alderspensjon etter kapittel 20
       */
    var tt_anv_AP_Kapittel20 = 0

    /*
       * tt anvendt brukt i uforepensjon
       */
    var tt_anv_UP = 0

    /*
       * trygdetid i prorata beregning
       */
    var prorataUP = 0.0

    /*
       * teller for proratabrok
       */
    var prorataUPTeller = 0

    /*
       * nevner for proratabrok
       */
    var prorataUPNevner = 0

    /*
       * personens gjeldende uttaksgrad
       */
    var uttaksgrad = 0

    /*
       * personens gjeldende uforegrad
       */
    var uforegrad = 0

}